wrappers/java.md

```diff
@@ -131,3 +131,9 @@ boolean isValid = new LicenseGate("YOUR_USER_ID")
 // Having trouble? Enable debug mode
 new LicenseGate("YOUR_USER_ID").debug().verify("LICENSE_KEY");
 ```
+
+#### Using your own LicenseGate server
+
+```java
+new LicenseGate("YOUR_USER_ID").setValidationServer("https://example.com").verify("LICENSE_KEY");
+```
```

Comment:
I'd add something like:

The URL should point to your LicenseGate backend server, not the web interface.

Reply:
Done, additionally changed the "setValidationServer" example to `backend.example.com` to further specify it to be the backend.

I have also expanded the RSA-Key example to include further information on not to strip the beginning and end-sections of the key.